includes/admin/meta-boxes/views/html-order-item.php

```diff
@@ -115,6 +115,10 @@
 			$tax_item_id       = $tax_item->get_rate_id();
 			$tax_item_total    = isset( $tax_data['total'][ $tax_item_id ] ) ? $tax_data['total'][ $tax_item_id ] : '';
 			$tax_item_subtotal = isset( $tax_data['subtotal'][ $tax_item_id ] ) ? $tax_data['subtotal'][ $tax_item_id ] : '';
+
+			$round_at_subtotal = 'yes' === get_option( 'woocommerce_tax_round_at_subtotal' );
```

Comment:
I am not sure if we also need to round here, because I think we need to preserve whatever the current value is unless this is explicitly edited. For egs, I may have a price of 2.555 (from an older order when round at subtotal setting was ticked), and I might edit that order. There is a possibility that I will save changes before giving much attention that this value is changed. 

Also, we anyway round the value before displaying it, so this might not be necessary. What do you think?

Reply:
Hm, it does seem to me like it would be slightly better to preserve the current value.

But what this change was meant to prevent is a similar (inverse?) scenario, reproducible in current master with the setting unchecked (as by default), described in the PR body: "clicking edit on the items and immediately saving reduces the amount by $0.01" due to the setting being respected at checkout but not on the order screen.

Do you see any possible way to solve both, so that it respects the current setting value on actual edit but at the same time doesn't affect the total when there is no edit?